core/modules/qdisp/Executive.cc

```diff
@@ -210,7 +214,7 @@ void Executive::markCompleted(int jobId, bool success) {
                 throw Bug(msg);
             }
         }
-        LOGS(_log, LOG_LVL_ERROR, "Executive: error executing jobId=" << jobId
+        LOGS(_log, LOG_LVL_ERROR, "Executive: error executing QId" << jobId
```

Comment:
I went through and replaced all of them with proper queryId and jobIds and got rid of the places where the executive was identified by `<< this <<` in log messages